repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl

```diff
@@ -18,46 +17,33 @@
 <#-- Content Page Pixel -->
 <@contentPixel document.getCanonicalUUID() document.title></@contentPixel>
 
+<#assign alphabetical_hash = group_blocks(flat_blocks(apiCatalogueLinks true))/>
 
-<article class="article article--filtered-list api-catalogue">
-
-    <@headerBanner document />
-
-    <div class="grid-wrapper grid-wrapper--article">
-        <div class="grid-row">
-            <div class="column column--two-thirds column--reset">
-                <#-- [FTL-BEGIN] 'Summary and optional Body' sections -->
-                <div id="section-summary" class="article-section article-section--summary  no-border">
-                    <div class="grid-row">
-                        <div class="column column--reset">
-                            <div class="rich-text-content">
-                                <#if document.body?has_content??>
-                                    <@hst.html hippohtml=document.body contentRewriter=gaContentRewriter/>
-                                </#if>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-                <#-- [FTL-END] 'Summary and optional Body' sections -->
-            </div>
-        </div>
+<@headerBanner document />
 
-        <#assign alphabetical_hash = group_blocks(flat_blocks(apiCatalogueLinks true))/>
+<div class="nhsd-t-grid nhsd-!t-margin-top-6">
+
+    <div class="nhsd-t-row">
+        <div class="rich-text-content nhsd-t-col-12">
+            <#if document.body?has_content??>
+                <@hst.html hippohtml=document.body contentRewriter=gaContentRewriter/>
```

Comment:
The `brContentRewriter` content rewriter should be used here to correctly style CMS content.

Reply:
Thanks Lee, have replaced it.